src/Plot/Matplotlib.luna

```diff
@@ -130,7 +168,36 @@ class Histogram:
         self.toChart.+ that
 
     def toJSON: self.toChart.toJSON
-
+    
+    # Saves the chart to an image file.
+    #
+    # Image will have dimensions as requested in function arguments. 
+    # The image file format is deduced from filename extension. Supported formats: eps, pdf, pgf, png, ps, raw, rgba, svg, svgz.
+    #
+    # > import Dataframes.Column
+    # > import Dataframes.Types
+    # > import Dataframes.Table
+    # >
+    # > def main:
+    # >     l1 = [1,23,78,4,12,7,89,6,7,19,17,13,15,56,89,23,0,1,45,30]
+    # >     col1 = Column.fromList "col1" Int64Type l1
+    # >     table = Table.fromColumns [col1]
+    # >     plot1 = table.chart.histogram "col1"
```

Comment:
I think we support just `table.histogram "col1"` ? (withou `.char`)
(unless we want to be extra explicit — but then we should be in other snippets as well)